website/templates/base.mako

```diff
@@ -246,6 +246,12 @@
     <%include file="alert.mako"/>
 </%def>
 
+## Default nav bar service values
+<%def name="service_title()">HOME</%def>
+<%def name="service_href()">/</%def>
+<%def name="service_support()">/support/</%def>
```

Comment:
These names could be more consistent.

How about `service_name`, `service_url`, and `service_support_url`?